src/views/CampusSafety/views/LostAndFound/views/MissingItemCreate/components/confirmReport/index.tsx

```diff
@@ -65,6 +67,14 @@ const ConfirmReport: React.FC<ConfirmReportProps> = ({ formData, onEdit, onSubmi
               <strong>Description:</strong> {formData.description}
             </Typography>
           </Grid>
+          {/* Conditionally render the stolen description if it exists */}
+          {formData.stolenDescription && (
+            <Grid item xs={12}>
+              <Typography>
+                <strong>Stolen Description:</strong> {formData.stolenDescription}
+              </Typography>
+            </Grid>
+          )}
```

Comment:
I think its because the report stolen button sends the stores the stolen description data once the person clicks on report stolen, i'll have to figure out a way to work around that. But honestly, i'm not sure why someone would report something stolen and right then go back to change that from being stolen, but yh its something that i guess needs to work better